javascript/src/utils/api/index.js

```diff
@@ -1,15 +1,37 @@
 import axios from "axios"
 import config from "./../../config_react.json";
+
 const getConfig = key => config["configReact"][key]
 
 const getCookie = (name) => {
-    var pattern = RegExp(name + "=.[^;]*")
-    var matched = document.cookie.match(pattern)
-    if(matched){
-        var cookie = matched[0].split('=')
-        return cookie[1]
+  var pattern = RegExp(name + "=.[^;]*")
+  var matched = document.cookie.match(pattern)
+  if (matched) {
+    var cookie = matched[0].split('=')
+    return cookie[1]
+  }
+  return false
+}
+
+const deleteCookie = (name, path, domain) => {
+  if (getCookie(name)) {
+    document.cookie = name + "=" +
+      ((path) ? ";path=" + path : "") +
+      ((domain) ? ";domain=" + domain : "") +
+      ";expires=Thu, 01 Jan 1970 00:00:01 GMT";
+  }
+}
+
+const handleError = (error) => {
+  if (error.response.status == 401
+      && error.response.headers['x-authenticated'] == "false") {
+      deleteCookie('idtoken', '/', window.location.hostname)
+      deleteCookie('atoken', '/', window.location.hostname)
+      deleteCookie('userinfo', '/', window.location.hostname)
+      // XXX it would be better if the root was / and not a path
+      //     Find a way to redirect to dashboard/root
+      window.location.href = `${window.environment}/${window.tenant}`
```

Comment:
this is right? or should be inverted? `${window.tenant}/${window.environment}`?

Reply:
done